mammoth/model_builder.py

```diff
@@ -366,12 +362,15 @@ def build_only_enc(model_opts, src_emb, task_queue_manager):
     """Truly only builds encoder: no embeddings"""
     encoder = build_encoder(model_opts, src_emb, task_queue_manager)
     if model_opts.param_init != 0.0:
-        for p in encoder.parameters():
-            p.data.uniform_(-model_opts.param_init, model_opts.param_init)
+        for name, p in encoder.named_parameters():
+            if not ("embedding" in name and "pe" not in name and model_opts.enable_embeddingless is True):
```

Comment:
just out of curiosity: why do we now make sure that the embeddings are not `pe`? was that an issue before

Reply:
This piece of code make sure that the 1-hot embedding vectors are not randomly initialized. This makes sure that PE embeddings are initialised too.

Reply:
right right, i read it wrong :P